src/CoreWCF.WebHttp/src/CoreWCF/Dispatcher/HttpStreamFormatter.cs

```diff
@@ -0,0 +1,113 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+using CoreWCF.Channels;
+using CoreWCF.Description;
+
+namespace CoreWCF.Dispatcher
+{
+    internal class HttpStreamFormatter : IDispatchMessageFormatter, IClientMessageFormatter
+    {
```

Comment:
I don't think you need to implement IClientMessageFormatter

Reply:
This was addressed in d6719a44244040967ed8059e2881340dae0cccac.